ptero_lsf/implementation/backend.py

```diff
@@ -219,6 +219,12 @@ def update_job_status(self, job_id):
 
                 LOG.exception("Exception occured while converting lsf"
                     " job to dictionary")
+
+                if (service_job.failed_update_count >= MAX_FAILS):
```

Comment:
MAX_FAILS was used in this module before this PF.  The get_job_ids_to_update() function uses it to constrain which jobs should have a status update scheduled.